calendarsubscription/calendarsubscription.go

```diff
@@ -46,3 +50,47 @@ func (cs CalendarSubscription) Normalize() (*CalendarSubscription, error) {
 
 	return &cs, nil
 }
+
+func (cs CalendarSubscription) renderICalFromShifts(shifts []oncall.Shift, reminderMinutes []int, scheduleName string) ([]byte, error) {
+	type iCalOptions struct {
+		Shifts          []oncall.Shift `json:"s,omitempty"`
+		ReminderMinutes []int          `json:"r,omitempty"`
+		ScheduleName    string         `json:"n,omitempty"`
+	}
+
+	var iCalTemplate = `
+		BEGIN:VCALENDAR
+		VERSION:2.0
+		PRODID:-//GoAlert//Calendar Subscriptions//EN
```

Comment:
Hmm, should we include the version? Also I think the cal-sub bit is redundant (since it's already a calendar).
>       ;Any text that describes the product and version
>       ;and that is generally assured of being unique.

Maybe this should be `//GoAlert//{{.Version}}//EN` (the current version of GoAlert, like in page source).

It would help if we generated bogus iCal data with a specific version, for example.